### ISSUE TITLE
Issue with Added/Changed/Deprecated directives in docstring

### ISSUE THREAD

**Issue by albertvillanova** [huggingface/doc-builder#266]
When using `<Added version="2.4.0" />` inside a docstring, it does not render anything.

See, e.g.: https://moon-ci-docs.huggingface.co/docs/datasets/pr_4767/en/package_reference/main_classes#datasets.Dataset.add_faiss_index
- Arg: `batch_size`

Docstring is:
```
            batch_size (Optional :obj:`int`): Size of the batch to use while adding vectors to the FaissIndex. Default value is 1000.

                <Added version="2.4.0" />

            train_size (Optional :obj:`int`):
```

Issue found in:
- huggingface/datasets#4767

Related to:
- #233
- #243

CC: @mishig25

**Comment by albertvillanova** [comment]
If docstring is instead:
```
            batch_size (Optional :obj:`int`): Size of the batch to use while adding vectors to the FaissIndex. Default value is 1000.

                <Added version="2.4.0"/>

            train_size (Optional :obj:`int`):
```
the rendering shows the word `undefined`:
```
batch_size (Optional int) — Size of the batch to use while adding vectors to the FaissIndex. Default value is 1000.

Added in 2.4.0
undefined

train_size (Optional int) — If the index needs a training step, specifies how many vectors will be used to train the index.

**Comment by mishig25** [comment]
The issue is solved with #268 & #269

You can checkout at: https://huggingface.co/docs/datasets/main/en/package_reference/main_classes#datasets.Dataset.add_faiss_index.batch_size
<img width="592" alt="Screenshot 2022-08-01 at 11 35 57" src="https://user-images.githubusercontent.com/11827707/182120526-81cd374f-0624-4296-a3f4-6cbf6920f345.png">

@albertvillanova thanks so much for reporting this bug 😊
